[2022-09-16 11:27:35,062] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: preprocessing_data.clean_data scheduled__2022-09-15T16:21:24.165110+00:00 [queued]>
[2022-09-16 11:27:35,067] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: preprocessing_data.clean_data scheduled__2022-09-15T16:21:24.165110+00:00 [queued]>
[2022-09-16 11:27:35,067] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-09-16 11:27:35,067] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-09-16 11:27:35,067] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-09-16 11:27:35,092] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): clean_data> on 2022-09-15 16:21:24.165110+00:00
[2022-09-16 11:27:35,095] {standard_task_runner.py:52} INFO - Started process 7217 to run task
[2022-09-16 11:27:35,099] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'preprocessing_data', 'clean_data', 'scheduled__2022-09-15T16:21:24.165110+00:00', '--job-id', '312', '--raw', '--subdir', 'DAGS_FOLDER/model_dag.py', '--cfg-path', '/tmp/tmpfthptu8q', '--error-file', '/tmp/tmp2ftcy_5_']
[2022-09-16 11:27:35,099] {standard_task_runner.py:80} INFO - Job 312: Subtask clean_data
[2022-09-16 11:27:35,146] {task_command.py:371} INFO - Running <TaskInstance: preprocessing_data.clean_data scheduled__2022-09-15T16:21:24.165110+00:00 [running]> on host rc1b-dataproc-m-xzxwmqcudfo0foh0.mdb.yandexcloud.net
[2022-09-16 11:27:35,201] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=preprocessing_data
AIRFLOW_CTX_TASK_ID=clean_data
AIRFLOW_CTX_EXECUTION_DATE=2022-09-15T16:21:24.165110+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-15T16:21:24.165110+00:00
[2022-09-16 11:29:04,334] {credentials.py:1094} INFO - Found credentials in environment variables.
[2022-09-16 11:29:06,712] {python.py:173} INFO - Done. Returned value was: None
[2022-09-16 11:29:06,720] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=preprocessing_data, task_id=clean_data, execution_date=20220915T162124, start_date=20220916T112735, end_date=20220916T112906
[2022-09-16 11:29:06,768] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-16 11:29:06,796] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
